katrain/core/game.py

```diff
@@ -601,6 +601,10 @@ def analyze_extra(self, mode, **kwargs):
         if mode == "game":
             nodes = self.root.nodes_in_tree
             only_mistakes = kwargs.get("mistakes_only", False)
+            move_range = kwargs.get("move_range", False)
+            if move_range:
```

Comment:
one thought: maybe this could be 
move_range is either absent/None or (start,end)

Reply:
Not sure what you mean - would the call in popups.kv be like this?
```
root.katrain("analyze-extra", "game", visits=visits.input_value, move_range=(start_move.input_value, end_move.input_value) if move_range.input_value else None, mistakes_only=mistakes.input_value)
```

Reply:
yep

Reply:
I changed it to that style.